README.md

```diff
@@ -58,85 +58,38 @@ The version listed is used in current environment. This must not be identical to
-        ```
-        Stopping talktalk-node_prisma_1 ... done
-        ```
-    - `$ docker-compose rm prisma mysql`
-        ```
-        Going to remove talktalk-node_prisma_1
-        Are you sure? [yN] y
-        Removing talktalk-node_prisma_1 ... done
-        Removing talktalk-node_mysql_1  ... done
-        ```
-
-    #### Debug: Check docker container's inspect
-    - `$ docker inspect [Mysql Docker Container's Name]`
-    - You can check container's name on `$ docker ps`
-
-    #### Debug: Check docker container log
-    - `$ docker-compose logs prisma`
-
-    #### Setting for connecting to Mysql Workbench
-    - append or change `ports` option in mysql service at `docker-compose.yml`
-      ```
-      services:
-        prisma:
-          environment:
-            PRISMA_CONFIG: |
-              databases:
-                default:
-                  connector: mysql
-                  host: mysql
-                  user: root
-                  password: prisma
-                  rawAccess: true
-                  port: 3306
-                  migrations: true
-        mysql:
-          image: mysql:5.7
-          restart: always
-          environment:
-            MYSQL_ROOT_PASSWORD: prisma
-          volumes:
-            - mysql:/var/lib/mysql
-          ports:            <- add this
-            - "33061:3306"  <- add this
-      volumes:
-        mysql:
-      ```
-    - kill and restart docker container
-    - Go to Workbench and Add MySQL Connections with this info
-      - Hostname: **0.0.0.0**
-      - Port: **33601**
-      - Username: **root**
-      - Password: **prisma**
+
+First of all, run `cp .env.sample .env` to generate environment file.
+
+Basically you need to change nothing but only `JWT_SECRET` with any value for local dev environment.
+
+| Name                         | Description               | required? | default               |
+|:-----------------------------|:--------------------------| --------- | --------------------- |
+| JWT_SECRET                   | Secret key to enc/dec JWT | true      |                       |
+| DEBUG                        | Flag to enable debug mode |           | false                 |
+| PRISMA_ENDPOINT              | Prisma endpoint URL       | true      | http://localhost:4466 |
+| PRISMA_MANAGEMENT_API_SECRET | Prisma service secret, required for production | false |  |
+| PRISMA_DB_CONNECTOR          | Database connector        | true      | mysql                 |
+| PRISMA_DB_DATABASE           | Database name             | true      | prisma                |
+| PRISMA_DB_USER               | Database username         | true      | prisma                |
+| PRISMA_DB_PASSWORD           | Database password         | true      | prisma                |
+
+Then,
+
+1. `docker-compose up -d`  
+   Run all prisma services in background.
+
+2. `npm run prisma:deploy`  
+   Initialize or apply changes of prisma data modeal to database.
```

Comment:
@cometkim `prisma:deploy` script is not in `package.json` now. do you have any idea to use this script in the future? 
and the word`modeal` in **Initialize or apply changes of prisma data modeal to database.** maybe a typo.
This content looks much better except what I pointed out!

Reply:
Thanks @geoseong , I just fixed typo and script.

just `prisma deploy` looks good to me, we already have a guide to install prisma command in above section

Reply:
@cometkim ok I got it. I checked your modified content and looks perfect.